scss/_patterns_pricing-block.scss

```diff
@@ -0,0 +1,48 @@
+@import 'settings';
+
+@mixin vf-p-pricing-block {
+  .p-pricing-block,
+  .p-pricing-block--25-75,
+  .p-pricing-block--50-50 {
+    @extend %vf-grid-row-equal-height;
+
+    .p-pricing-block__tier {
+      &:before {
+        border-top: 1px solid $colors--theme--border-default;
+        content: '';
+        margin-bottom: calc($spv--small - 1px);
+        width: calc(100% + map-get($grid-gutter-widths, default));
+      }
+
+      &:last-child::before {
+        width: 100%;
+      }
+
+      @media (max-width: $threshold-4-6-col) {
+        grid-column: span 4;
+
+        &:not(:last-child)::before {
+          width: 100%;
+        }
+      }
+    }
+  }
+
+  .p-pricing-block--25-75 {
+    @media (min-width: $threshold-6-12-col) {
+      .p-pricing-block__tier:first-child {
+        grid-column: 3 / span 2;
```

Comment:
Use grid settings to calculate the span rather than hard-coding the `2` here

Reply:
Can the '3' remain as is? Or is it preferred to d o something like `calc($grid-8-columns / 4 + 1)`?

Reply:
Hm, yes I would use the variable there as well